anybar/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for anybar 2.0.1
+// Project: https://github.com/sindresorhus/anybar
+// Definitions by: Chris Khoo <https://github.com/khoomeister>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'anybar' {
```

Comment:
ok updated @vvakame - i looked at a few others and chose what i thought was right. lmk if still not good - thx!